### ISSUE TITLE
Add elgentos/masquerade to Phar.io

### ISSUE THREAD

**Issue by peterjaap** [phar-io/phar.io#79]
Preferred alias; `masquerade`

https://github.com/elgentos/masquerade

**Comment by theseer** [comment]
Thank you for your alias registration request.
Unfortunately, we only allow aliases for _signed releases_.

So before we can add the requested alias, please ensure you created a signed release. Some help on how to do that can be found here: https://phar.io/howto/sign-and-upload-to-github.html.

Once that is done, we'll gladly register your alias.

Until then, if you want to install your phar using phive anyhow, you'd have to use `phive install --force-accept-unsigned elgentos/masquerade`.

**Comment by peterjaap** [comment]
@theseer thanks for the info. I have a refactor coming up for the tool, when I finish that I'll sign it as well and create a new issue here.
